cypress/integration/happy-path-with-accessibility.js

```diff
@@ -146,9 +146,27 @@ describe('Get help to retrain smoke test', function() {
     checkAccessibility();
   });
 
-  it('should allow me to find local schemes and offers', function() {
+  it('should allow me to get CV advice', function() {
     cy.go('back')
-    cy.contains('Show me local offers').click()
+    cy.contains('Get CV advice').click()
+    checkAccessibility();
+  });
+
+  it('should allow me to get cover letter advice', function() {
```

Comment:
the cover letter advice step is repeated :)

Reply:
deleted.